cfpb/cfgov-refresh
# [WIP] Paying for College - Costs app functionality

This PR will add full functionality to the "College Costs" application in Paying for College. The user can navigate the app, enter amounts, make choices, and see results. Some bugs and poor styling remain - this is a work in progress, currently.

## Additions
- Add remaining dispatcher files and functionality
- Add remaining model files and functionality
- Add remaining view files and functionality
- Add remaining util files and functionality
- Add code to allow user to enter and calculate financial values
- Add code to calculate debt
- Add interactive code to buttons, etc
- Add popstate handling to allow for 'Back' and save functionality

## How to test this PR
1. Pull it down and get cfgov-refresh running locally
2. Create a college-costs page in Wagtail and check it out.
3. The landing page will be empty, but hit "Get Started" and see if it functions as you would expect.

## Checklist

_[Feel free to delete any checkboxes that are not applicable to this PR.]_

  - PR titles are used to generate the change log in [releases](../../releases),
    so good ones make that easier to scan.
  - Consider prefixing with the feature/section you're addressing,
    e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"
  - Potential places for documentation:
    - [This repo’s docs](https://cfpb.github.io/cfgov-refresh/) (edit the files in the `/docs` folder) –
      for basic, close-to-the-code documentation on working with this repo
    - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
    - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing

_[When new or significantly modified front-end functionality is present,
  the following things should be tested.
  Feel free to delete this section if not applicable to this PR.]_

#### Browser testing

Visually tested in the following supported browsers:

[Further guidance on browser support](https://github.com/cfpb/development/blob/master/guides/browser-support.md)

#### Accessibility

- [ ] Does not introduce new errors or warnings in
      [WAVE](https://wave.webaim.org/extension/)

#### Other
